edb/schema/links.py

```diff
@@ -311,6 +311,24 @@ def _cmd_tree_from_ast(
         assert isinstance(cmd, sd.Command)
         return cmd
 
+    def get_ast_attr_for_field(
+        self,
+        field: str,
+        astnode: Type[qlast.DDLOperation],
+    ) -> Optional[str]:
+        if (
+            field == 'required'
+            and issubclass(astnode, qlast.CreateConcreteLink)
+        ):
+            return 'is_required'
```

Comment:
Maybe we finally can drop `is_`?

Reply:
Yes, I have that on my TODO list, but keep procrastinating on it.  Will tackle that soon.